src/java/org/apache/cassandra/tools/nodetool/ListSnapshots.java

```diff
@@ -64,15 +63,17 @@ public void execute(NodeProbe probe)
                 return;
             }
 
-            final long trueSnapshotsSize = probe.trueSnapshotsSize();
             TableBuilder table = new TableBuilder();
             // display column names only once
-            final List<String> indexNames = snapshotDetails.entrySet().iterator().next().getValue().getTabularType().getIndexNames();
+            List<String> indexNames = snapshotDetails.entrySet().iterator().next().getValue().getTabularType().getIndexNames();
+            indexNames.subList(0, indexNames.size() - 1);
 
             if (includeEphemeral)
-                table.add(indexNames.toArray(new String[indexNames.size()]));
-            else
                 table.add(indexNames.subList(0, indexNames.size() - 1).toArray(new String[indexNames.size() - 1]));
+            else
```

Comment:
the below logic deals with the fact that we do not want to have another column of "Raw true size" in the output of `nodetool listsnapshots`. All we are interested in are the numbers we sum up internally and display that in `TrueDiskSpaceUsed` at then end of the output.

We do have true sizes already returned, but they are in _human friendly format_ and we can not sum these up.